src/components/index.ts

```diff
@@ -0,0 +1,11 @@
+import NavTab from "./NavTab.vue"
+
+const components = [NavTab]
+
+import { App } from "vue"
+
+export default {
+  install: (app: App) => {
+    components.forEach((c) => app.component(c.name!, c))
```

Comment:
declaring c.name! is questionable. Is there a better way to handle potential missing name property?